content/en/docs/ops/configuration/traffic-management/traffic-routing/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Understanding Traffic Routing
+linktitle: Traffic Routing
+description: How Istio routes traffic through the mesh.
+weight: 30
+keywords: [traffic-management,proxy]
+owner: istio/wg-networking-maintainers
+test: n/a
+---
+
+One of the goals of Istio is to act as a "transparent proxy" that can be dropped in place into an existing cluster and allow traffic to continue to function as previously.
+However, there are some important ways that Istio causes traffic to be handled differently than a typical Kubernetes cluster, due to additional features (such as request load balancing) and architectural constraints.
+To understand what is happening in your mesh, it is important to understand how Istio routes traffic.
+
+{{< warning >}}
+This document describes low level implementation details. For a higher level overview, check out the traffic management [Concepts](/docs/concepts/traffic-management/) or [Tasks](/docs/tasks/traffic-management/).
+{{< /warning >}}
+
+## Protocols
+
+Unlike Kubernetes, Istio has the ability to process higher level protocols such as HTTP and TLS.
+In general, there are three classes of protocols Istio understands:
+
+* HTTP, which includes HTTP/1.1, HTTP/2, and gRPC. Note that this does not include TLS encrypted traffic (HTTPS).
+* TLS, which includes HTTPS.
+* Opaque TCP bytes.
+
+Which protocol is used is determined by [protocol selection](/docs/ops/configuration/traffic-management/protocol-selection/).
+
+The use of "TCP" can be confusing, as in other contexts it is used to distinguish between other L4 protocols, such as UDP.
+When referring to the TCP protocol in Istio, this typically means we are treating it as an opaque stream of bytes,
+and not parsing higher level constructs such as TLS or HTTP.
+
+## Traffic Routing
+
+When the Istio proxy receives a request, it must decide where to forward the request to, if anywhere.
+By default, this will be the original service that was request, unless [customized](/docs/tasks/traffic-management/traffic-shifting/).
```

Comment:
```suggestion
By default, this will be to the original service that was requested, unless [customized](/docs/tasks/traffic-management/traffic-shifting/).
```

that link is only to a page that talks about a certain use case; I think the others all apply too?  Maybe a link to the general concepts page?